MSW/MSW/Watchdog.cs

```diff
@@ -110,6 +110,17 @@ public void UpdateAccessTokens()
 				}
 			}
 
+			public void Tick()
+			{
+				Task.Run(RequestTick);
```

Comment:
This could be a lambda ```
() =>
{
	Console.WriteLine("Watchdog server tick on session token {0}", m_currentAccessToken.GetTokenAsString());
	APIRequest.Perform(ServerApiRoot, "/api/game/Tick", m_currentAccessToken.GetTokenAsString(), null);
}```